client/src/components/CreatePost.vue

```diff
@@ -73,72 +56,59 @@ export default {
         [{ 'indent': '-1' }, { 'indent': '+1' }],
         [{ 'direction': 'rtl' }],
         [{ 'header': [1, 2, 3, 4, 5, 6, false] }]
```

Comment:
@devesh-verma  is there aa better way to do this?

Reply:
The is the suggested way by vue2-editor plugin, for adding custom options in editor